swan/python/jupyter_swan_config.py

```diff
@@ -0,0 +1,35 @@
+import os
+#from swancontents import get_templates
+
```

Comment:
Misses `contents_manager_class` and `checkpoints_class`

Reply:
Ignore `checkpoints_class`.

Set `contents_manager_class` like this:
`c.ServerApp.contents_manager_class = 'swancontents.filemanager.SwanEosFileManager'`